docs/user-manual/consume.md

```diff
@@ -37,15 +90,18 @@ machine-readable JSON and then logs various details about the Datum as it goes.
 It's very easy to customize the configuration options of this script to match
 the requirements of your own dApp.
 
-It is made freely available under an open-source Apache v2.0 license so that
-you can test, integrate, and extend your own solutions using the world's most
-[trustworthy/trustless](verify) authenticated fact statement feeds, provided on
-the world's most decentralized and
+This resource has been made freely available under under an open-source Apache
```

Comment:
If im reading this right we have double - "under under"

Reply:
Oops, fixed!